packages/factory_reset_tools/lib/dbus/factory_reset.dart

```diff
@@ -98,10 +100,17 @@ final List<BootOption> defaultBootOption = [
   ),
 ];
 
-List<BootOption> getResetOptions({String path = defaultFilePath}) {
+List<BootOption> getResetOptions({String path = ''}) {
+  var fpath = path;
```

Comment:
No abbreviations, and this code block should be done in a function so that it doesn't have to be repeated

Reply:
Also, it's better to use null as the default here than an empty string

Reply:
For 'null', I tried, but it complains that it can't be assigned to a String...

Reply:
Ok, now I now what happened: I was mixing go and dart 😅

Reply:
Done.